maxim/logger/writer.py

```
@@ -12,10 +12,23 @@
 import time
 import uuid
 from concurrent.futures import ThreadPoolExecutor
+from http.client import RemoteDisconnected
 from queue import Queue
 from typing import Optional
 
 import filetype
+from requests.exceptions import (
+    ConnectionError as RequestsConnectionError,
+    ConnectTimeout,
+    HTTPError,
```

Ruff (0.11.9)

`requests.exceptions.HTTPError` imported but unused

Remove unused import

(F401)